swift/apple/Firezone/Assets.xcassets/LogoText.imageset/Contents.json

```diff
@@ -72,6 +77,7 @@
           "value" : "dark"
         }
       ],
+      "filename" : "welcomeview-dark 2.svg",
```

Comment:
So these identical SVGs are for different scales? Does it need them to be duplicated files if they're all SVGs?

Reply:
Unfortunately not. When you select the image in the Assets editor it creates duplicates. Should compress really well anyway so probably not increasing app bundle too much.